### ISSUE TITLE
Iphone Safari not rendering correctly

### ISSUE THREAD

**Issue by Santoshah** [WebDevSimplified/netflix-sliding-movie-ui#1]
When I try to view this on my iphone 12 Safari browser, the width of the boxes gets very small. I think the width is not able to calculate properly in case of mobile safari. However it seem working fine with chrome and other browser. Any idea how to fix this ?

**Comment by GODZ-k** [comment]
yeah, correct this layout only works on Netflix like movie posters. when I resize the poster or image it does not work properly.
I had to solve this problem. if you should wanna. I can send you. @Santoshah
